lib/3scale/backend/server/falcon.rb

```diff
@@ -14,16 +16,11 @@ def self.start(global_options, options, args)
           manifest = global_options[:manifest]
           return unless manifest
 
-          argv = ['falcon']
-          argv_add argv, true, '--bind', 'http://0.0.0.0'
-          argv_add argv, options[:port], '--port', options[:port]
-
-          # Starts the prometheus server if needed. Just once even when spanning
-          # multiple workers.
-          argv_add argv, true, '--preload', 'lib/3scale/prometheus_server.rb'
+          ENV["PORT"] = options[:port] if options[:port]
```

Comment:
The port comes from `3scale_backend` script, and it's not directly reachable from the new config file `falcon.rb`. So I'm setting this env variable here to be read from the config file. Ugly but effective.

Reply:
One problem I see with this is that it might be confusing to any user that they need (or can) set the port through the env var.
But if somebody sets both `PORT=3001` and provide `-p 3000`, the former will not be effective.

Ideally, we would have just one place where we'd set the port, and reuse it. But I understand that the flow is a bit clunky, and we can't easily fetch in `falcon.rb` what is passed as command arguments to `3scale_backend` :/

So, I'm fine with this.

Reply:
Thanks.

Just replacing this line by 

```suggestion
          ENV["PORT"] |= options[:port]
```

will give preference to the env variable, if you think that's better.

Reply:
To be honest, I'm not sure what is better :grimacing: 
But I think having a possibility of relying on `PORT` could be nice.

Reply:
I've seen that the `-p` parameter is always being provided. It's enforced by the `3scale_backend` script, and set to `-p 3000` by default. So `PORT` env variable is always going to be ignored unless we give it priority. I pushed some changes: https://github.com/3scale/apisonator/pull/427/commits/7aa1a03f5402b2e5e6fa6ccadb6b04b2a9bcbac8